backend/views/home.pug

```diff
@@ -38,6 +38,15 @@ block content
 				.col.s12.m4.l4.center-align
 					img.responsive-img(src="stylesheets/img/network.png")
 			.divider
+			.row.valign-wrapper
+				.col.s12.m8.l8
+					h5 Finding Crimes from Images 
+					p In addition to getting crimes from databases, we also want to dynamically find crimes from Twitter and other sites. 
+					p This means that instead of having to feed data in to the application manually we can simply take the images automatically.
+					p After we've gotten the images, we feed them into Amazon Rekognition to find tags, and then use a co-training model from the images, captions and Rekognition data to determine whether crimes are occuring in the image.
```

Comment:
I'd Say `after we've obtained the images, we use them as data for Amazon Rekognition to find tags. Then we use a trained hybrid model to make use of this data along with manually annotated captions to determine whether or not crimes are appearing in images, along with crime severity.`

Reply:
Done.